src/screens/Ticketing/Tickets/use-recent-tickets.ts

```diff
@@ -207,7 +212,7 @@ export default function useRecentTickets() {
 
   useEffect(() => {
     fetchRecentFareContracts();
-  }, []);
+  }, [fareContracts]);
```

Comment:
Suggest changing to array-length, so we don't get unecessary updates

```suggestion
  }, [fareContracts.length]);
```

Reply:
Good point. Maybe we could use `[fareContracts[0]?.id]` to also eliminate the (very few) cases where logging in/out to a user with equal ticket count?

Reply:
Hmm, good point! 🤔 That's probably a good idea. Another way (so the way the array sort would not matter) would be to evaluate the latest `created` time. Something like `Math.max(0, ...fareContracts.map((a) => a.created.seconds));` But not sure if we're over-analyzing the whole thing then?